### PR TITLE
Update Kitty UI

### PR DESCRIPTION
**Pull request by Kswin01**
Shuffle Kitty UI around:

IMG_1354

### CODE REVIEW

File: examples/kitty/client/kitty.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 reader_stream = None
 writer_stream = None
 
-IP_ADDRESS = "172.16.0.2"
+IP_ADDRESS = "10.13.1.17"
```
Comment (alwin-joshy) [inline examples/kitty/client/kitty.py:21]: Not sure if this change should be included

Comment (Kswin01) [reply examples/kitty/client/kitty.py]: Thanks for catching that!

File: examples/kitty/client/kitty.py [hunk 2]
```diff
@@ -54,26 +54,24 @@ def show(self):
         fb.machine_fb_send(self.buf, self.width, self.height)
 
 
-# @alwin: all the vars highlighted are still (especially) dodgy
 ######################################################
```
Comment (alwin-joshy) [inline examples/kitty/client/kitty.py:57]: Maybe get rid of the ########### thing and move these up with the rest of the global variables

**Review by alwin-joshy — APPROVED** [review]

File: examples/kitty/client/kitty.py [hunk 3]
```diff
@@ -54,26 +54,24 @@ def show(self):
...
     global writer_stream
     while True:
-        # @alwin: is it really necessary to have this in a try-catch?
         try:
             info(f"Sending heartbeat at {time.time()}")
             writer_stream.write(b'200 0 0\n')
             await writer_stream.drain()
         except OSError as e:
             if (e.errno == errno.ECONNRESET):
+                info("ERROR: RESETTING CONNECTION")
```
Comment (Ivan-Velickovic) [inline examples/kitty/client/kitty.py:74]:
```suggestion
                error("RESETTING CONNECTION")
```

File: examples/kitty/client/kitty.py [hunk 4]
```diff
@@ -82,20 +80,19 @@ def heartbeat():
...
-        # @alwin: is it really necessary to have this in a try-catch?
         try:
             writer_stream.write(
                 bytes(f'100 {token} {''.join('{:02x}'.format(x) for x in card_id)} 1.0' + '\n', 'utf-8'))
-            # @alwin: Should this have an await()?
             await writer_stream.drain()
             break
         except OSError as e:
             if (e.errno == errno.ECONNRESET):
+                info("ERROR: RESETTING CONNECTION")
```
Comment (Ivan-Velickovic) [inline examples/kitty/client/kitty.py:95]: Same thing as my previous comment.

**Review by Ivan-Velickovic — APPROVED** [review]

**Pull request merged**